cura/BuildVolume.py

```diff
@@ -570,8 +573,17 @@ def _computeDisallowedAreasPrime(self, border_size, used_extruders):
         machine_width = self._global_container_stack.getProperty("machine_width", "value")
         machine_depth = self._global_container_stack.getProperty("machine_depth", "value")
         for extruder in used_extruders:
-            prime_x = extruder.getProperty("extruder_prime_pos_x", "value") - machine_width / 2 #Offset by half machine_width and _depth to put the origin in the front-left.
-            prime_y = machine_depth / 2 - extruder.getProperty("extruder_prime_pos_y", "value")
+            prime_x = extruder.getProperty("extruder_prime_pos_x", "value")
+            prime_y = - extruder.getProperty("extruder_prime_pos_y", "value")
+
+            #Ignore extruder prime position if it is not set
+            if prime_x == 0 and prime_y == 0:
```

Comment:
It probably isn't, but I could not find a good way to check if the setting was actually set in the machine definition. 

Perhaps a better way to handle this would be to make the default value for extruder_prime_pos_x and _y something like -1000000 (can we do -INF in default_values?).